benchmarks/nemo/nemolite2d/manual_versions/regent/read_config.rg

```diff
@@ -0,0 +1,56 @@
+
+local c = regentlib.c
+local cstring = terralib.includec("string.h")
+
+--Terra list containing a list of values we need.
+local config_fields_tiles = terralib.newlist({
+  --Tilesize options
+  {field = "t1", type = int64, default_value = 256, cmd_line = "-t1"},
+  {field = "t2", type = int64, default_value = 512, cmd_line = "-t2"}
+})
+
+config = terralib.types.newstruct("config")
+config.entries:insertall(config_fields_tiles)
+
+--Taken from pennant_common.rg
```

Comment:
I'm not sure if I can do this while meeting the Apache license rules (since I don't want to add an extra license). I assume I should probably rewrite this code?

Reply:
Oh that's a tricky question, I think it is small and general enough to be fair use but to be in the safe side it may be better to rewrite it as you say.

Reply:
It is admittedly difficult to rewrite as I'm not really sure what one might change from this function. @rupertford any idea regarding this? I think the Apache v2 license is used.